app/services/page_updater.rb

```diff
@@ -81,9 +81,10 @@ def refresh?
 
   def update_page
     return unless @params[:page]
+    @page.slug = nil
```

Comment:
This will indeed create a new slug for the page. We'll need to either use the history module or revert back several commits when we still had our own `create_slug` method. Does it matter for our ActionKit integration that the slug of the page might change? @EricBoersma I think you're best aware of what problems that might bring up.